sklearn/utils/tests/test_validation.py

```diff
@@ -10,7 +10,11 @@
 import scipy.sparse as sp
 
 from sklearn.utils.testing import assert_true, assert_false, assert_equal
-from sklearn.utils.testing import assert_raises, assert_raises_regexp
+from sklearn.utils.testing import (
```

Comment:
right it on 2 lines and don't import assert_raises_regexp because it is the same than assert_raises_regex. FInd and replace if you see it.
``` python 
from sklearn.utils.testing import assert_raises
from sklearn.utils.testing import assert_raises_regex
```